yogstation/code/modules/clothing/donator.dm

```diff
@@ -189,11 +189,20 @@
 	name = "Inmate belongings box"
 	desc = "A criminals' belongings, returned upon their release. Just stop killing people."
 
-/obj/item/storage/box/boxta/PopulateContents() 
+/obj/item/storage/box/boxta/PopulateContents()
 	. = ..()
 	new /obj/item/clothing/neck/falcon(src)
 	new /obj/item/clothing/head/fedora/gtrim_fedora(src)
 
+/obj/item/storage/box/somecustodian
+	name = "Golden Box"
```

Comment:
since this is apparently a cheap toy this should be improper

Reply:
Piss